fuzzers/074-dump_all/jobnodes.tcl

```diff
@@ -0,0 +1,47 @@
+set blocknb [lindex $argv 0]
+set start [expr int([lindex $argv 1])]
+set stop [expr int([lindex $argv 2])]
+
+create_project -force -part $::env(XRAY_PART) $blocknb $blocknb
+set_property design_mode PinPlanning [current_fileset]
+open_io_design -name io_1
+
+#set_param tcl.collectionResultDisplayLimit 0
+set_param messaging.disableStorage 1
+
+set root_fp [open "root_node_${blocknb}.csv" w]
+
+set nodes [get_nodes]
+
+for {set j $start } { $j < $stop } { incr j } {
+
+    set node [lindex $nodes $j]
+    puts "New node ! - $j"
```

Comment:
Remove puts to stdout (here and in jobtiles)

Reply:
Should be good now :)